staging/src/k8s.io/kubectl/pkg/cmd/create/create_job.go

```diff
@@ -216,13 +218,17 @@ func (o *CreateJobOptions) Run() error {
 		if err != nil {
 			return fmt.Errorf("failed to create job: %v", err)
 		}
+	} else {
```

Comment:
Same as comment above for create_cronjob.go

Reply:
I just realized create deploy is getting refactored too.  I think my comment above still stands though.  Try to make it look like this one, IMO:  https://github.com/kubernetes/kubernetes/pull/91199/files

Reply:
Same as above...